### PR TITLE
dura: add service

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Added service

### CODE REVIEW

**Review by carlocab — DISMISSED** [review]

File: Formula/dura.rb [hunk 1]
```diff
@@ -27,6 +27,14 @@ def install
     system "cargo", "install", *std_cargo_args
   end
 
+  service do
+    run [opt_bin/"dura", "serve"]
+    keep_alive true
+    error_log_path var/"log/dura.log"
+    log_path var/"log/dura.log"
```
Comment (carlocab) [inline Formula/dura.rb:34]: Should this be different from the error path, since it seems to produce JSON?

Comment (ankane) [reply Formula/dura.rb:34]: We could separate them. Is there a convention we should use? Looking at other formulas, seems like for errors, we could do `dura.err`, `dura.err.log`, `dura-error.log`, or `dura/error.log`.

Comment (ankane) [reply Formula/dura.rb:34]: cc @tkellogg

Comment (carlocab) [reply Formula/dura.rb:34]: @tkellogg's example plist uses `dura.log.json`. I guess we can use that.

Comment (ankane) [reply Formula/dura.rb]: Updated both filenames to match the example plist (`dura.log.json` and `dura.stderr.log`).

**Review by carlocab — APPROVED** [review]

**Comment by ankane** [comment]
Thanks!

**Pull request merged**